webApp.js

```diff
@@ -1,18 +1,16 @@
 import express from 'express';
-import {AsyncLocalStorage, executionAsyncId} from 'async_hooks';
+import {storage} from './storage.js'
 import * as uuid from 'uuid';
 import router from './router.js'
 
 let app = express();
-let storage = new AsyncLocalStorage();
 
 global.kResourceStore = storage.kResourceStore;
 
-app.use(async (req, res, next) => {
-    await storage.run(new Map(), () => {
+app.use((req, res, next) => {
+    storage.run(new Map(), () => {
```

Comment:
Even if `run` has some asynchronous magic, it does not return a promise. calling `await` on it does not make sense

Reply:
Here, we call run once per request, but `new` only once.